LibreNMS/__init__.py

```diff
@@ -183,6 +184,12 @@ def normalize_wait(seconds):
     return ceil(seconds - (time() % seconds))
 
 
+def reset_signals():
+    # https://bugs.python.org/issue38435
+    # https://bugs.python.org/issue32985
+    signal.signal(signal.SIGCHLD, signal.SIG_DFL)
```

Comment:
You are running this in preexec_fn, but I've seen other places it is just run in the parent process.  Just double checking this is what we want.